### PR TITLE
add GetServiceEndpoint

### PR DESCRIPTION
**Pull request by ekaterinaryzhenkova**

### CODE REVIEW

File: src/AdminService.Business/Commands/GetServiceEndpointCommand.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    private readonly IResponseCreator _responseCreator;
+
+    public GetServiceEndpointCommand(
+      IServiceConfigurationRepository configurationRepository,
+      IServiceEndpointRepository repository,
+      IEndpointInfoMapper mapper,
+      IResponseCreator responseCreator)
+    {
+      _configurationRepository = configurationRepository;
+      _endpointRepository = repository;
```
Comment (ndoriomedov) [inline src/AdminService.Business/Commands/GetServiceEndpointCommand.cs]: rename to endpointRepository

**Review by ndoriomedov — APPROVED** [review]

**Review by asyagraf — APPROVED** [review]

**Review by VladSpitsin — APPROVED** [review]

**Review by grinakate — APPROVED** [review]

File: src/AdminService.Data/Interfaces/IServiceConfigurationRepository.cs [hunk 2]
```diff
@@ -17,5 +17,7 @@ public interface IServiceConfigurationRepository
     Task<List<Guid>> EditAsync(List<Guid> servicesIds);
 
     Task<bool> DoesAppInstalledAsync();
+
+    Task<DbServiceConfiguration> GetServiceAsync(Guid serviceId);
```
Comment (MariyaSin) [inline src/AdminService.Data/Interfaces/IServiceConfigurationRepository.cs]: you are in the serviceConfigurationRepository, it is clear that you get Service data, rename it to GetAsync

File: src/AdminService.Data/ServiceConfigurationRepository.cs [hunk 3]
```diff
@@ -92,5 +92,19 @@ public async Task<bool> DoesAppInstalledAsync()
       return (await _provider.ServicesConfigurations
         .FirstAsync()).ModifiedAtUtc != default;
     }
+
+    public async Task<DbServiceConfiguration> GetServiceAsync(Guid serviceId)
```
Comment (MariyaSin) [inline src/AdminService.Data/ServiceConfigurationRepository.cs]: same

File: src/AdminService.Data/ServiceConfigurationRepository.cs [hunk 4]
```diff
@@ -92,5 +92,19 @@ public async Task<bool> DoesAppInstalledAsync()
...
         .FirstAsync()).ModifiedAtUtc != default;
     }
+
+    public async Task<DbServiceConfiguration> GetServiceAsync(Guid serviceId)
+    {
+      DbServiceConfiguration dbServiceConfiguration = await _provider.ServicesConfigurations
+        .Include(x => x.Endpoints)
+        .FirstOrDefaultAsync(x => x.Id == serviceId);
+
+      if (dbServiceConfiguration is null)
```
Comment (MariyaSin) [inline src/AdminService.Data/ServiceConfigurationRepository.cs]: remove the condition

File: src/AdminService/Controllers/ServiceEndpointController.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+using Microsoft.AspNetCore.Mvc;
+
+namespace LT.DigitalOffice.AdminService.Controllers
+{
+  [Route("[controller]")]
+  [ApiController]
+  public class ServiceEndpointController : ControllerBase
+  {
+    [HttpGet("get")]
+    public async Task<OperationResultResponse<ServiceEndpointsInfo>> SomeGet(
```
Comment (MariyaSin) [inline src/AdminService/Controllers/ServiceEndpointController.cs]: SomeGet - strange name))

**Review by MariyaSin — APPROVED** [review]

**Pull request merged**